icepyx/__init__.py

```diff
@@ -1 +1 @@
-from .core import query
+from icepyx.core.query import Query
```

Comment:
Wow, this was a lot easier than I thought! I still see some `from icepyx import is2class as ipd` code under the 'icepyx/tests' folder though, could you please change them to use the new 'import icepyx as ipx' syntax?